packages/block-library/src/paragraph/edit.native.js

```diff
@@ -16,13 +16,15 @@ function ParagraphBlock( {
 	mergeBlocks,
 	onReplace,
 	setAttributes,
-	style: oldStyle,
+	wrapperProps,
+	globalStyle,
+	style,
 } ) {
-	const { align, content, placeholder, style } = attributes;
+	const { align, content, placeholder } = attributes;
 
 	const styles = {
-		...oldStyle,
-		color: style && style.color && style.color.text,
+		...style,
+		color: wrapperProps?.style?.color || globalStyle?.color,
```

Comment:
Looking at this line I'm assuming that at the moment `color` is the only global style and wrapper prop that we have available in mobile?

Reply:
ATM we support color, backgroundColor, and gradient. Other props will be added as well when they land on the web because we use the web logic

Reply:
@SergioEstevao 
I have checked that and we have 3 kinds of styles in wrapper props at this moment.
- lineHeight - `__experimentalLineHeight`
- fontSize - `__experimentalFontSize`
- color/backgroundColor/gradient  - `__experimentalColor`

Unfortunately, it seems like the `lineHeight` doesn't work in the rich-text at all. The fontSize works only for a paragraph block but not for heading block - It's related to the`tagName` that is passed to the `rich-text`.

So, we have these props already in the `wrapperProps` but our `edit` implementations don't use them yet.

Reply:
BackgroundColor does not work in the Paragraph block?

Reply:
Yeah, the backgroundColor works. I should mention that in my prev comment that I refactored it a bit :)